### ISSUE TITLE
Allow moving the search bar

### ISSUE THREAD

**Issue by goldeyegold** [emilyploszaj/emi#587]
In some chest mods they have chests that are so big that the hop bar is under the search bar it would be nice if there was a setting to move the search bar to the right

**Comment by emilyploszaj** [comment]
There is one, "center search bar"

**Comment by goldeyegold** [comment]
I've been trying to figure out if you could do this or not I suggest making a little more clear in the config menu on what that setting does

**Comment by xKOBAYASHIMARUx** [comment]
Super glad I found this but yeah this option isn't obvious whatsoever. I found this thread by googling, which brought me to the EMI discord (from what I've seen there is no user wiki, just a dev wiki and a modpack dev wiki), the discord server managed to link back to this github issue but for whatever reason google couldn't, then bingo.

The setting really shouldn't be a true/false option. Makes it super not clear what that config does. For anyone else searching for the answer, setting "center search bar" config to true puts it in the bottom center (self explanatory) and setting to false puts it into the bottom right (this is the option that's not clear).
